### PR TITLE
fix(fb pixel): add default action source

### PR DESCRIPTION
**Pull request by Gauravudia**
## Description of the change

> 
- actions_source parameter is required in [Facebook Pixel](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/server-event#action-source)
- if action_source parameter is not present in the request. By default we will set it as '`other`'

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/facebook_pixel/transform.js [hunk 1]
```diff
@@ -246,7 +246,10 @@ const responseBuilderSimple = (message, category, destination, categoryToContent
     if (!isActionSourceValid) {
       throw new InstrumentationError('Invalid Action Source type');
     }
+  } else {
+    commonData.action_source = 'other';
```
Comment (saikumarrs) [inline src/v0/destinations/facebook_pixel/transform.js:250]: Add documentation reference. Also, some explanation about this choice.

**Review by saikumarrs — CHANGES_REQUESTED** [review]

**Review by ItsSudip — APPROVED** [review]

File: src/v0/destinations/facebook_pixel/transform.js (hunk 1 above)
Comment (Gauravudia) [reply src/v0/destinations/facebook_pixel/transform.js:250]: We will do the follow-up task on that to correctly configure the action source (for JS sdk, mobile and other)

Comment (saikumarrs) [reply src/v0/destinations/facebook_pixel/transform.js:250]: Could you add some details about the fix in the PR?

**Review by shrouti1507 — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

File: src/v0/destinations/facebook_pixel/transform.js (hunk 1 above)
Comment (Gauravudia) [reply src/v0/destinations/facebook_pixel/transform.js:250]: added

**Pull request merged**